lib/node-utils/is-node-of-type.ts

```diff
@@ -59,3 +59,4 @@ export const isReturnStatement = ASTUtils.isNodeOfType(
 export const isFunctionExpression = ASTUtils.isNodeOfType(
   AST_NODE_TYPES.FunctionExpression
 );
+export const isIdentifier = ASTUtils.isNodeOfType(AST_NODE_TYPES.Identifier);
```

Comment:
The `isIdentifier` util already exists in the `ASTUtils` exported from `@typescript-eslint/utils`.

It's misleading to find these AST utils imported from 2 different modules, so we should re-export all the default ones from our custom module, and forbid the `@typescript-eslint/utils`. Until then, let's used the one exposed from `@typescript-eslint/utils`